php-fpm/recipes/cloudwatch.rb

```diff
@@ -1,6 +1,7 @@
 
 cronscript = "#{node['php-fpm']['prefix']}/bin/phpfpm-cloudwatch.sh"
 
+include_recipe 'python'
```

Comment:
@gilleyj see `awscli/recipes/default.rb` — I think it includes python. And then you don't need to list this "dependency" in "php-fpm".